pepper-apis/src/test/java/org/broadinstitute/ddp/route/ReceiveDsmNotificationRouteTest.java

```diff
@@ -327,8 +330,10 @@ public void testEvents_setInitialKitSentTime() {
                 assertNotNull(record.getInitialKitSentTime());
             });
         } finally {
-            TransactionWrapper.useTxn(handle ->
-                    DBUtils.checkDelete(1, handle.attach(KitScheduleSql.class).deleteRecord(recordId)));
+            TransactionWrapper.useTxn(handle -> {
+                DBUtils.checkDelete(1, handle.attach(KitScheduleSql.class).deleteRecord(recordId));
+                DBUtils.checkDelete(1, handle.attach(KitConfigurationDao.class).deleteConfiguration(kitConfigId.get()));
```

Comment:
Some of the kit tests will fail depending on the test order, and the cause seems to be that we're not cleaning up the kit configuration setup here.